src/laboratory/utils_base_unit.py

```diff
@@ -0,0 +1,41 @@
+from laboratory.models import BaseUnitValues
+
+# from laboratory.utils_base_unit import get_conversion_units
+
+
+def get_base_unit(unit):
+    query = BaseUnitValues.objects.filter(measurement_unit=unit)
+    if query.exists():
+        unit = query.first().measurement_unit_base
+        return unit
+
+def get_conversion_units(unit, amount):
+    query = BaseUnitValues.objects.filter(measurement_unit=unit)
+    if query.exists():
+        unit = query.first().measurement_unit_base
```

Comment:
You can create a variable for unit element like 

unit = query.first()
base = unit.measurement_unit_base
value = unit.si_value